genie-web/src/main/java/com/netflix/genie/web/configs/JobConfig.java

```diff
@@ -192,4 +197,28 @@ public WorkflowTask jobKickoffTask(
             registry
         );
     }
+
+    /**
+     * Create a {@link ProcessChecker.Factory} suitable for UNIX systems.
+     *
+     * @param executor       The executor where checks are executed
+     * @param jobsProperties The jobs properties
+     * @return a {@link ProcessChecker.Factory}
+     */
+    @Bean
+    @Autowired
```

Comment:
You don't need `@Autowired` on `@Bean` functions

Reply:
I just kept consistent with the "style" of the rest of this file...
I'll remove all unnecessary `@Autowired`